src/main/scala/mesosphere/marathon/core/health/impl/MarathonHealthCheckManager.scala

```diff
@@ -155,8 +164,7 @@ class MarathonHealthCheckManager(
         // remove health checks for which the app version is not current and no tasks remain
         // since only current version tasks are launched.
         for {
-          (version, activeHealthChecks) <- ahcs(appId)
-          if version != app.version && !activeAppVersions.contains(version)
+          (version, activeHealthChecks) <- ahcs(appId) if version != app.version && !activeAppVersions.contains(version)
```

Comment:
not very familiar with the scala syntax used before your patch but I'm not sure to get what your change is here.

Reply:
nothing. I will revert it. Probably intellij playing with the syntax.